src/redux/sagas/offices.js

```diff
@@ -79,6 +80,16 @@ function* registerCompanyworker(action) {
   }
 }
 
+function* getAddressLLMWorker(action) {
+  try {
+    const llmBalance = yield call(getAddressLLM, action.payload.walletAddress);
```

Comment:
rename to getAddressCentralizedLLM or something

Reply:
or to make sure that everytime its called its called like centralizedBackend.getLLMs or whatever